### PR TITLE
Removed homestead bootnodes

### PR DESCRIPTION
**Pull request by tiabc**
Remove homestead bootnodes from `cht.json`. Their existence reflected the reality incorrectly as we don't have any homestead bootnodes in our cluster.

### CODE REVIEW

File: geth/params/config_test.go [hunk 1]
```diff
@@ -357,11 +357,7 @@ var loadConfigTestCases = []struct {
 			require.True(t, nodeConfig.BootClusterConfig.RootNumber >= 805)
 
 			enodes := nodeConfig.BootClusterConfig.BootNodes
-			expectedEnodes := []string{
-				"enode://93833be81c3d1bdb2ae5cde258c8f82ad1011a1bea8eb49fe50b0af394d4f7f7e45974356870552f36744efd732692a64865d1e8b64114eaf89a1bad0a1903a2@51.15.64.29:30303",
-				"enode://d76854bc54144b2269c5316d5f00f0a194efee2fb8d31e7b1939effd7e17f25773f8dc7fda8c4eb469450799da7f39b4e364e2a278d91b53539dcbb10b139635@51.15.73.37:30303",
-				"enode://57874205931df976079e4ff8ebb5756461030fb00f73486bd5ec4ae6ed6ba98e27d09f58e59bd85281d24084a6062bc8ab514dbcdaa9678fc3001d47772e626e@51.15.75.213:30303",
-			}
+			expectedEnodes := []string{}
 			require.Equal(t, expectedEnodes, enodes)
```
Comment (themue) [inline geth/params/config_test.go]: Only a question: Is this one of the cases which should be `require.Equal(t, enodes, expectedNodes)`?

File: static/bindata.go [hunk 2]
```diff
@@ -120,12 +120,12 @@ func scriptsWeb3Js() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "scripts/web3.js", size: 497689, mode: os.FileMode(420), modTime: time.Unix(1507576052, 0)}
+	info := bindataFileInfo{name: "scripts/web3.js", size: 497689, mode: os.FileMode(420), modTime: time.Unix(1507747781, 0)}
```
Comment (themue) [inline static/bindata.go]: Expecting these times to be proper defined.

File: geth/params/config_test.go (hunk 1 above)
Comment (tiabc) [reply geth/params/config_test.go]: As you can see, it's even hard to distinguish when it's already been written.
But no, the order is correct.

File: static/bindata.go (hunk 2 above)
Comment (tiabc) [reply static/bindata.go]: Sorry?

File: geth/params/config_test.go (hunk 1 above)
Comment (themue) [reply geth/params/config_test.go]: Thanks, I do know other libs with different order. Where the returned value is compared to an expected one. `foo := 123; assert.Equal(foo, 123)`. That's why I'm asking.

File: static/bindata.go (hunk 2 above)
Comment (themue) [reply static/bindata.go]: The unix times are changed here and it's not directly obvious to me where this value is come from.

Comment (tiabc) [reply static/bindata.go]: `make generate`

Comment (themue) [reply static/bindata.go]: Ah, top, thanks. 👍

**Pull request merged**